tokio-util/Cargo.toml

```diff
@@ -41,9 +41,9 @@ futures-core = "0.3.0"
 futures-sink = "0.3.0"
 futures-io = { version = "0.3.0", optional = true }
 futures-util = { version = "0.3.0", optional = true }
-log = "0.4"
 pin-project-lite = "0.2.0"
 slab = { version = "0.4.4", optional = true } # Backs `DelayQueue`
+tracing = "0.1.31"
```

Comment:
since there's only logging in `codec`, I wonder if we want the `tracing` dep to be optional, so it's only pulled in with `codec`?